File: Data Structures & Algorithms/maximum-subarray/submission-0.java
class Solution {
    public int maxSubArray(int[] nums) {
        int n = nums.length; 
        int maxSum = Integer.MIN_VALUE;

        for(int i = 0; i < n; i++) {
            int currentSum = 0;
            for(int j = i; j < n; j++) {
                currentSum += nums[j];
                maxSum = Math.max(currentSum, maxSum);
            }
        }
        return maxSum;
    }
}

File: Data Structures & Algorithms/single-number/submission-0.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        for i in range(len(nums)):
            is_unique = True
            for j in range(len(nums)):
                if i != j and nums[i] == nums[j]:
                    is_unique = False
                    break
            if is_unique:
                return nums[i]
